### PR TITLE
Update default base url and docs

### PR DESCRIPTION
**Pull request by kdelee**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-PYTEST_OPTIONS = -vvv
-CLOUDIGRADE_USER=''
+PYTEST_OPTIONS :=-vvv
```
Comment (elyezer) [inline Makefile]: Any specific reason for using non recursive variables?

File: scripts/oc-auth.sh [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+export CLOUDIGRADE_BASE_URL="${CLOUDIGRADE_BASE_URL:-test.cloudigra.de}"
 export CLOUDIGRADE_USER=$(uuidgen --random)
```
Comment (elyezer) [inline scripts/oc-auth.sh]: According to the README:

```
located in ``scripts/oc-auth.sh`` which will use ``CLOUDIGRADE_USER`` or create
unique name using ``uuidgen`` and create a superuser, and then retreive an
```

According to README, this line should reuse a previously set CLOUDIGRADE_USER env var, does'n it? If it should, then it should be updated to do what it is saying on README.

File: Makefile (hunk 1 above)
Comment (kdelee) [reply Makefile]: No really I was fiddling around thinking about indicating a default driver and then changed my mind.

File: scripts/oc-auth.sh (hunk 2 above)
Comment (kdelee) [reply scripts/oc-auth.sh]: updated readme

File: Makefile (hunk 1 above)
Comment (kdelee) [reply Makefile]: got rid of changes since they were not needed

**Pull request merged**